tests/integration/state.go

```diff
@@ -175,6 +175,9 @@ type state struct {
 	// nodes.
 	docIDs [][]client.DocID
 
+	// CIDs by index, by field name, by doc index, by collection index.
```

Comment:
todo: `by field name` should say `by field id`. I was wondering why you where asserting `field name` with value `"1"` and `"2"` until I realized it's the `field id` that is used.